org.mixedrealitytoolkit.input/Readers/PinchInputReader.cs

```diff
@@ -145,16 +141,14 @@ protected virtual void OnDisable()
         /// </summary>
         protected virtual void Update()
         {
-            if (m_firstUpdate)
-            {
-                UpdateActionValidCaches();
-                m_firstUpdate = false;                
-            }
+            m_isTrackingStatePolyfilled = trackedPoseDriver.GetIsPolyfillDevicePose();
+            m_isSelectPolyfilled = !selectAction.action.HasAnyControls();
+            m_isSelectValuePolyfilled = !selectActionValue.action.HasAnyControls();
 
             // Workaround for missing select actions on devices without interaction profiles
             // for hands, such as Varjo and Quest. Should be removed once we have universal
```

Comment:
Note: Varjo uses Ultraleap hand tracking (on devices that support hand tracking, which isn't all Varjo devices). Ultraleap supports the hand interaction profile.

Reply:
Comment updated!